examples/tour/src/main.rs

```diff
@@ -591,6 +591,26 @@ impl<'a> Step {
                     .width(Length::Fill)
                     .horizontal_alignment(alignment::Horizontal::Center),
             )
+            .push(Space::with_height(Length::Units(25)))
+            .push(Text::new(
+                "Images can also be used as background for other items:",
+            ))
+            .push(
+                Container::new(
+                    Container::new(
+                        Column::new()
+                            // We're cheating here. Since Background::Image does not (yet?) have advanced resizing options, let's include content that has roughly the same aspect ratio as the background image
+                            .push(Text::new("A first line"))
+                            .push(Text::new("A second line"))
+                            .push(Text::new("A third line")),
```

Comment:
Given the current implementation, I believe quads and triangle meshes will still be renderered under the image background. You can try pushing a `Button` here to see the issue.

As I mentioned in #1235, there is a bunch of limitations in our rendering pipelines that make background images hard.

Reply:
Hmmm, that's a nice catch.

![image](https://user-images.githubusercontent.com/6873361/155486265-8225b0c0-f877-41f7-ae00-9f5817e4e31c.png)

I'm not sure I'm knowledgeable enough to help here (graphical rendering is not my cup of tea). But maybe I can have a look if you're mentoring me or tell me "generally speaking" what should be done (and where).

Otherwise, do you think the first commits (that move `image::Handle` to `iced_core`) are worth being merged into master, even if we drop the (broken) commit that adds background images? Would this slight architectural change would benefit to the rest of us?